CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Version 5.0.0
```

Comment:
I think examples would be very helpful to understand the breaking nature of these two changes.